app/_hub/kong-inc/kafka-log/1.3-x.md

```diff
@@ -1,6 +1,7 @@
 ---
 name: Kafka Log
 publisher: Kong Inc.
+version: 1.3-x # 0.1.0 internal handler
```

Comment:
is there a way to make the internal handler version show up in the hub, next to the plugin version menu? or is it better to just leave it as a comment?

Reply:
> is there a way to make the internal handler version show up in the hub, next to the plugin version menu? or is it better to just leave it as a comment?

That's a great question (I think this no because this is a yaml file but I can test)  but we do need to determine exactly what dictates a plugin version. Internal handler seems preferable to me and less confusing. Incrementing a plugin version based on KE/CE release number causes bloat in the doc source and is also confusing as it doesn't match the handler version number. cc: @eskibars

Reply:
It does work but I'm not sure if it adds to the confusion. Tested on other Kafka plugin.

![image](https://user-images.githubusercontent.com/3756245/108519871-b0fa1000-728f-11eb-8a0b-8601a9f724ab.png)

Reply:
can't I see it in the deploy preview... the version should be the internal handler only, not based on the KE/CE release number IMO

Reply:
> can't I see it in the deploy preview... the version should be the internal handler only, not based on the KE/CE release number IMO

Agree versions should be internal handler only. Not sure how to handle all the legacy versions based on KE/CE. I'd prefer to just have the latest version only, but @eskibars will have to advise and approve.

I tested it locally on the other Kafka preview, so you wouldn't see it in this PR's preview. It just adds to the confusion IMO.